frontend/smart-kickers-game/src/apis/goalManipulation.js

```diff
@@ -1,8 +1,17 @@
 import axios from 'axios';
 import config from '../config';
 
-export function pointsManipulation(teamID, action) {
-  axios.post(`${config.apiBaseUrl}/goal?action=${action}&team=${teamID}`).catch((err) => {
-    alert(err);
-  });
+export async function pointsManipulation(teamID, action) {
+  try {
+    let result = await axios.post(`${config.apiBaseUrl}/goal?action=${action}&team=${teamID}`);
```

Comment:
Please, use `const` if the value of the variable isn't changed later. It improves readability, because at a first glance it gives an information whether the variable is changed later or not.

Reply:
Ok. Done